salt/metalk8s/internal/preflight/mandatory.sls

```diff
@@ -39,22 +39,8 @@ Set sysctl {{ item }} value to {{ value }}:
     - require:
       - kmod: Add the module br_netfilter to kernel
     {%- endif %}
-    - require_in:
-      - cmd: Apply sysctl configuration files
 {%- endfor %}
 
-# This is needed to ensure values loaded from files match in-memory ones
-Apply sysctl configuration files:
-  cmd.run:
-    - name: sysctl --system
-
-Check sysctl are well configured:
-  metalk8s_checks.run:
-    - name: metalk8s_checks.sysctl
```

Comment:
So, should we remove this `metalk8s_checks.sysctl` method? Or would it still be useful somehow (e.g. from command-line)?

Reply:
I don't know, that's why I didn't remove it.
We may want to use this at some point if we use post install checks or maybe not.

Reply:
OK fine by me, the code is tested, so we can keep it.